internal/policies/mount/adsys_mount/Cargo.toml

```diff
@@ -11,11 +11,11 @@ name = "lib"
 path = "src/lib.rs"
 
 [dependencies]
-anyhow = "1.0.65"
-chrono = "0.4.22"
```

Comment:
For minor unstable versions, I would keep the exact version.

Reply:
You mean as in `gio = "0.16.x` rather than `gio = "0.16"`?
I have no idea how cargo decides this, to be honest. If it only fetches the versions which have docs, I think this should be safe.

Reply:
I think this boils down to the upstream commitment to semantic versioning.

I think specifying `0.3` for the `time` crate is fine, and its upstream recommends doing that: https://time-rs.github.io/book/index.html.

As for `glib` and `gio` stuff, their documentation also seems to recommend specifying the minor version and omitting the patch version: https://gtk-rs.org/gtk-rs-core/stable/latest/docs/gio/.

Reply:
In this case, I would also rather have the version pinned. We can be a little more safer this way and keep some versioning consistency for the crates.